app/views/spree/admin/overview/_products.html.haml

```diff
@@ -12,10 +12,7 @@
     - if @product_count > 0
       %div.seven.columns.alpha.list-item
         %span.six.columns.alpha
-          - if @product_count == 1
-            = t(".active_products_count_single", product_count: @product_count )
-          - else
-            = t(".active_products_count_multiple", product_count: @product_count )
+          = t(".active_products", count: @product_count )
```

Comment:
You can use exactly the same code in line 19 and move the translation from `spree_admin_enterprises_producers_manage_products` to `spree.admin.overview.products.active_products.zero`.